src/Okta.Sdk/Generated/UserFactorResolver.Generated.cs

```diff
@@ -70,6 +65,11 @@ protected override Type GetResolvedTypeInternal(IDictionary<string, object> data
                 return typeof(HardwareUserFactor);
             }
             
+            if (value.Equals("token:hotp"))
+            {
+                return typeof(CustomHotpUserFactor);
+            }
+            
```

Comment:
We should consider updating the code generation templates to use a `Dictionary<string, Type>` instead of these `if` statements; we would get a micro performance improvement.